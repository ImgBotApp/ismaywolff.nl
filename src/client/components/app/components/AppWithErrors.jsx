import React from 'react'
import { string } from 'prop-types'
import { Container } from '../../../../shared/components/container'
import { BootError } from '../../../../shared/components/errors'

const AppWithErrors = ({ errorMessage }) => (
  <Container>
    <BootError errorMessage={errorMessage} />
  </Container>
)

AppWithErrors.propTypes = {
  errorMessage: string.isRequired
}

export default AppWithErrors
