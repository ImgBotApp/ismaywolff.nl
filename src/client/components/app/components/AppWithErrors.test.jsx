import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import AppWithErrors from './AppWithErrors'

describe('<AppWithErrors />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AppWithErrors errorMessage={'Message'} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
