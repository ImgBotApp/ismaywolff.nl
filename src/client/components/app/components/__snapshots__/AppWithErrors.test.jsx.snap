// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppWithErrors /> renders correctly 1`] = `
<styled.div>
  <BootError
    errorMessage="Message"
  />
</styled.div>
`;
